x-pack/solutions/security/plugins/security_solution/public/common/components/user_privileges/user_privileges_context.tsx

```diff
@@ -48,11 +51,17 @@ export const UserPrivilegesProvider = ({
   const crud: boolean = kibanaCapabilities[SECURITY_FEATURE_ID].crud === true;
   const read: boolean = kibanaCapabilities[SECURITY_FEATURE_ID].show === true;
 
-  const listPrivileges = useFetchListPrivileges(read);
-  const detectionEnginePrivileges = useFetchDetectionEnginePrivileges(read);
+  const rulesCapabilities = kibanaCapabilities[RULES_FEATURE_ID];
+  const readRules = rulesCapabilities?.[RULES_UI_READ] === true;
+  const editRules = rulesCapabilities?.[RULES_UI_EDIT] === true;
```

Comment:
Could it be the case `rulesCapabilities?.[RULES_UI_EDIT]` is true but `rulesCapabilities?.[RULES_UI_READ]` is not set? Logically the `read` capability is a subset of the `edit` capability.